packages/loader/container-loader/src/container.ts

```diff
@@ -1799,7 +1795,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     private async fetchSnapshotTree(specifiedVersion: string | undefined):
         Promise<{snapshot?: ISnapshotTree; versionId?: string}>
     {
-        const version = await this.getVersion(specifiedVersion ?? this.id);
+        const version = await this.getVersion(specifiedVersion ?? this._resolvedUrl?.id ?? null);
```

Comment:
Why not use this.getVersion(specifiedVersion ?? null);

I'd love to start moving toward not using resolve ID as a special thing to note head. If null does this, let's use it everywhere and slowly remove usage of ID here.

Reply:
because using it is failing the snapshot tests